### PR TITLE
make changes to parser

### PR DESCRIPTION
**Pull request by infoverload**
## Summary of the changes / Why this improves CrateDB

This is a draft PR for https://github.com/crate/crate/issues/2036

## Checklist

 - [ ] Added an entry in `CHANGES.txt` for user facing changes
 - [ ] Updated documentation & `sql_features` table for user facing changes
 - [x] Touched code is covered by tests
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed
 - [ ] This does not contain breaking changes, or if it does:
    - It is released within a major release
    - It is recorded in ``CHANGES.txt``
    - It was marked as deprecated in an earlier release if possible
    - You've thought about the consequences and other components are adapted
      (E.g. AdminUI)

### CODE REVIEW

**Comment by infoverload** [comment]
Could I get some feedback on the changes made to the parser?

File: libs/sql-parser/src/main/java/io/crate/sql/SqlFormatter.java [hunk 1]
```diff
@@ -952,7 +952,14 @@ public Void visitDropTable(DropTable<?> node, Integer indent) {
...
             }
-            node.table().accept(this, indent);
+            var tables = node.tables().iterator();
+            while (tables.hasNext()) {
+                tables.next().accept(this, indent);
+                if (tables.hasNext()) {
+                    builder.append(',');
+                }
+            }
+            builder.append(' ');
```
Comment (mfussenegger) [inline libs/sql-parser/src/main/java/io/crate/sql/SqlFormatter.java:962]: Does this leave a trailing whitespace? If so, maybe change to:

```suggestion
                    builder.append(', ');
                }
            }
```

File: libs/sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java [hunk 2]
```diff
@@ -576,7 +577,8 @@ public Node visitShowSessionParameter(SqlBaseParser.ShowSessionParameterContext
 
     @Override
     public Node visitDropTable(SqlBaseParser.DropTableContext context) {
-        return new DropTable((Table) visit(context.table()), context.EXISTS() != null);
+        var dropTable = new DropTable(Lists2.map(context.table(), x -> (Table) visit((ParseTree) x)), context.EXISTS() != null);
```
Comment (mfussenegger) [inline libs/sql-parser/src/main/java/io/crate/sql/parser/AstBuilder.java:580]:
```suggestion
        var dropTable = new DropTable(Lists2.map(context.table(), x -> (Table) visit(x)), context.EXISTS() != null);
```

**Review by mfussenegger — COMMENTED** [review]
Left two minor suggestions, otherwise the parser changes look good so far :+1:

**Comment by mfussenegger** [comment]
Are you still interested in working on this? Otherwise I'd close the PR for now, as it is outdated and looks stale

**Comment by mfussenegger** [comment]
Closing it for now. If you still feel like picking it up please do and create a new PR

**Pull request closed without merging**